### ISSUE TITLE
[Huobi] WatchOrderBook doesn't process feeds occasionally

### ISSUE THREAD

**Issue by krychla1** [ccxt/ccxt#10599]
Dear fellows,
I'cant get the huobi watchOrderBook work reliably. For every couple of subscriptions it doesn't process the feeds from ws. 

Please try this snippet multiple times or extend the symbols with additional ones. I am able to simulate the problem in 1 of 5-10 attempts.

```javascript

const ccxtpro = require('ccxt.pro');
let orderbooks = {}

async function tick(exchange) {
    try {
        const delay = 1000 // every 1000 ms
        while (true) {
            const symbols = Object.keys(orderbooks)
            // console.log(new Date(), exchange.id, 'orderbooks', symbols)
            for (let i = 0; i < symbols.length; i++) {
                const symbol = symbols[i]
                console.log(new Date(), symbol, orderbooks[symbol]['bids'][0], orderbooks[symbol]['asks'][0], orderbooks[symbol].nonce, ((Date.now() - orderbooks[symbol].timestamp) / 1000).toFixed())      //with ccxt.pro/js/Base/OrderBook.js line 95 this.timestamp = snapshot.timestamp || Date.now()
            }
            await exchange.sleep(delay)
        }
    } catch (e) {
        console.log(e);
    }
}

async function watchOrderBook(exchange, symbol) {
    if (!exchange.markets[symbol])
        return;
    while (true) {
        try {
            const orderbook = await exchange.watchOrderBook(symbol)
            orderbooks[symbol] = orderbook
        } catch (e) {
            console.log(e)
        }
    }
}

async function main() {
    const symbols = ["LINK/ETH"]

    const exchange = new ccxtpro[exchangeName](params);       //huobi
    await exchange.loadMarkets();

    const symbolsPromises = symbols.map(symbol => watchOrderBook(exchange, symbol))

    await Promise.all([symbolsPromises.concat(tick(exchange))])
}
```
ccxt ver: ccxt.pro@0.9.28 with ccxt@1.61.24

Please see the log attached for verbose. You will see listener registering feeds, however the orderbook isn't being updated.
[huobi_log.txt](https://github.com/ccxt/ccxt/files/7542061/huobi_log.txt)

**Comment by krychla1** [comment]
Hi @kroitor, any updates on this? Huobi watchorderbook is unusable in this state.

**Comment by kroitor** [comment]
@krychla1 will get to it very soon!

**Comment by kroitor** [comment]
Pardon for the long wait.
This issue has been addressed in CCXT Pro 1.0.19+
Please, let us know if you have similar issues after upgrading.
Thanks for your feedback!

**Comment by krychla1** [comment]
Seems working, switching on in production. Will get back here if there are any issues.
Thanks!
